apis/supervisor/idp/v1alpha1/types_tls.go.tmpl

```diff
@@ -3,9 +3,29 @@
 
 package v1alpha1
 
+
+// CABundleSource provides a source for CA bundle used for client-side TLS verification.
```

Comment:
All the same comments from the concierge version of this file also apply to this file.

Reply:
will be done in next push